### PR TITLE
Improved Brew search algorithm and OneUpdater

### PR DESCRIPTION
**Pull request by varp**

### CODE REVIEW

**Review by jeffbyrnes — COMMENTED** [review]
How might we see the “OneUpdater” additions or changes?

**Comment by varp** [comment]
Sorry, for the late answer. OneUpdater patch consists of two parts:
- info.plist file which is the workflow main file itself
- the script added to the workflow at https://github.com/willfarrell/alfred-pkgman-workflow/pull/156/commits/8b0c09eacf5de102e7d31eed680a0a61488c83df#diff-ef6dac0c1c122a5c9efd56197356cab3608e10a0ff362195e31e0bda9d4a1e7eR1505

Connections of the OneUpdate script and the rest of the WF components you can see by importing the new linked version of the Alfred package

One more thing, on every update a release engineer should update the workflow version and commit back ([Makefile target](https://github.com/willfarrell/alfred-pkgman-workflow/pull/156/commits/8b0c09eacf5de102e7d31eed680a0a61488c83df#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R23)) again the info.plist to the repo (with the new version) to OneUpdate could detect the new version.

File: info.plist [hunk 1]
```diff
@@ -0,0 +1,3267 @@
...
+		</dict>
+		<dict>
+			<key>config</key>
+			<dict>
+				<key>concurrently</key>
+				<false/>
+				<key>escaping</key>
+				<integer>0</integer>
+				<key>script</key>
+				<string># THESE VARIABLES MUST BE SET. SEE THE ONEUPDATER README FOR AN EXPLANATION OF EACH.
```
Comment (varp) [inline info.plist:1505]: The OneUpdater script

File: info.plist [hunk 2]
```diff
@@ -0,0 +1,3267 @@
...
+	<dict>
+		<key>XDEBUG_CONFIG</key>
+		<string>idekey=PHPSTORM remote_host=localhost remote_port=9000</string>
+	</dict>
+	<key>variablesdontexport</key>
+	<array>
+		<string>XDEBUG_CONFIG</string>
+	</array>
+	<key>version</key>
+	<string>4.2.0</string>
```
Comment (varp) [inline info.plist:3263]: The current WF version

File: info.plist (hunk 1 above)
Comment (jeffbyrnes) [reply info.plist:1505]: Because I’m not familiar, would you link to some docs on OneUpdater, and/or explain what it does?

**Comment by joeybaker** [comment]
hmmm… a quick search doesn't inspire a lot of confidence about OneUpdater: https://duckduckgo.com/?t=ffab&q=OneUpdater&atb=v171-1&ia=web 
image

This is hard to review since all the interesting code is in a binary 😬

File: info.plist (hunk 1 above)
Comment (varp) [reply info.plist:1505]: Of course , https://github.com/vitorgalvao/alfred-workflows/tree/master/OneUpdater

Comment (jeffbyrnes) [reply info.plist:1505]: Looks like this is the actual source code: https://github.com/vitorgalvao/alfred-workflows/blob/master/OneUpdater/source/source.sh

Quick glance, seems cool! I’ll grab the workflow & look in more detail soon.

**Comment by varp** [comment]
@jeffbyrnes Any progress on this?

**Comment by jeffbyrnes** [comment]
@varp unfortunately, you’ll need to re-do these changes against the latest version of `Package Managers.alfredworkflow`, since there’s now conflicts. I can’t resolve conflicts with a binary file like this.

**Comment by varp** [comment]
@jeffbyrnes Hello, I've updated PR, finally. Please review

**Comment by varp** [comment]
And the new one is coming 😊  https://github.com/varp/alfred-pkgman-workflow/pull/1

**Review by jeffbyrnes — APPROVED** [review]

**Comment by jeffbyrnes** [comment]
Ok I see how this all comes together now.

I’ll see about wiring this up so we automatically publish the AlfredWorkflow package as part of a GitHub Release.

**Comment by jeffbyrnes** [comment]
I asked over on varp/alfredpkgman-workflow#1, but reiterating here for others’ benefit:

> @varp perhaps you could open a new PR w/ some additional instructions in the README? I’m struggling a little to figure out how this build process goes now.

**Pull request merged**